gatk-launch

```diff
@@ -85,6 +85,10 @@ def main(args):
         runGATK(sparkRunner, dryRun, gatkArgs, sparkArgs)
     except GATKLaunchException as e:
         sys.stderr.write(str(e)+"\n")
+        sys.exit(3)
```

Comment:
slightly more information content than 1.. 

- 1 GATKException or Miscellaneous java error
- 2 UserException
- 3 Some error in gatk-launch itself